src/chains/algorand_1/examples/transfers.ts

```diff
@@ -38,12 +38,23 @@ const composeTokenTransferParams: Partial<AlgorandActionAssetTransferParams> = {
 }
 
 const composeAlgoPaymentParams: Partial<AlgorandActionPaymentParams> = {
-  from: 'VBS2IRDUN2E7FJGYEKQXUAQX3XWL6UNBJZZJHB7CJDMWHUKXAGSHU5NXNQ',
+  from: 'WCZV75K44NWDFWTYJCCOJ6NZC2TYAODLIC7GFMMXRKY2JNPW3LTB4IUL3U',
   to: 'TF6PJW7VSEKD5AXYMUXF5YGMPDUWBJQRHH4PYJISFPXAMI27PGYHKLALDY',
   note: 'transfer memo',
   amount: 1,
 }
 
+// requires transaction setup with betanet endpoints and accounts.
+// example is not repeatable. Each time the account(from) is successfully reKeyed,
+// next time transaction will require signature of the new address(reKeyTo)
+const composeAlgoReKeyParams: Partial<AlgorandActionPaymentParams> = {
```

Comment:
Can this be runnable multiple times? Examples should 'reset' so they can be run again.
Maybe you can write two sections for the function 
step 1 - rekey to account B - print something that shows the account key
step 2- rekey back to account A